app/controllers/pages_controller.rb

```diff
@@ -1,3 +1,5 @@
 class PagesController < ApplicationController
-  def guidelines; end
+  def guidelines
+    @guideline = SiteSetting.first
```

Comment:
Is SiteSetting only going to be for Guidelines? Will guidelines always be the first SiteSetting?

Reply:
SiteSetting will hold information regrading dynamic content nature for every part we want to make dynamic.
i-e: for now we are doing Guideline.
Yes it will have only one object that will change every time a change is made
Also we have added a rake task to add default Guideline content

Reply:
Please rename `SiteSettings` to `PageContents`.